app/src/main/java/com/github/geohunt/app/ui/components/leaderboard/LeaderboardListItem.kt

```diff
@@ -54,15 +51,15 @@ fun LeaderboardListItem(
             color = textColor
         )
 
-        ProfileIcon(user = user)
+        entry.displayIcon?.invoke()
```

Comment:
Why invoke when you can just do `entry.displayIcon?()` ?

Reply:
This sadly doesn't work 😢 😭 
You can't use something else than a method after ?